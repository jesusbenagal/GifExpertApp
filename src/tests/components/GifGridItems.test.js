import React from 'react';
import { shallow } from 'enzyme';

import GifGridItem from '../../components/GifGridItem';

describe('Pruebas para GifGridItem', () => {
    
    let wrapper = shallow( <GifGridItem />);

    beforeEach(() => {

        wrapper = shallow( <GifGridItem />);

    })

    test('Debe mostrar correctamente <GifGridItem/>', () => { 
        
        expect ( wrapper ).toMatchSnapshot();

     })

});
