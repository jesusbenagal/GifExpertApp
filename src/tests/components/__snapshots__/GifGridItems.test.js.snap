// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas para GifGridItem Debe mostrar correctamente <GifGridItem/> 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
